solutions/src/main/java/ru/sberbank/school/task06/Pet.java

```diff
@@ -0,0 +1,67 @@
+package ru.sberbank.school.task06;
+
+import java.util.Objects;
+
+public class Pet{
```

Comment:
WhitespaceAround: &#39;{&#39; is not preceded with whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1703843683>*